### ISSUE TITLE
Background script is unavailable

### ISSUE THREAD

**Issue by oifj34f34f** [translate-tools/linguist#401]
OS: Linux
Browser: Firefox

![](https://github.com/github-linguist/linguist/assets/128177553/2c683fa6-5b5b-448f-a227-474ec019e4a7)

This happens if browser history is disabled:

![](https://github.com/translate-tools/linguist/assets/128177553/50d391f2-5851-425f-9386-fa53139e2a5f)

**Comment by vitonsky** [comment]
No, we should not close issue. It is good you detect this bug. Probably some WebExtension API are limited when you change this option in browser. It is possible that Linguist even don't use actively this API, so we can just remove module that throw error and problem will be fixed
